Meridian59.DebugUI/ObjectEditor.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Windows.Forms;
+using Meridian59.Data.Models.AdminData;
+
+namespace Meridian59.DebugUI
+{
+    public partial class ObjectEditor : Form
+    {
+        public ObjectEditor(AdminObject obj)
```

Comment:
This is not completed yet. There are a lof of ways I can work better within your design/architecture. I'll work on that slowly. I feel like now that I have AdminData class in Meridian59 i Might need an AdminUIController that handles talking to AdminData.

I probably need to define some Interfaces for AdminObject and AdminProperty, INotifyProeprtyChanged is probably not going to work in the long run.